Src/Chill/TestBase.cs

```diff
@@ -94,7 +95,7 @@ internal void TriggerTest(Action testAction, bool expectExceptions)
             {
                 try
                 {
-                    testAction();
+                    testAction.ExecuteInDefaultSynchronizationContext();
```

Comment:
Given that we do this here, in the base class. There is no need to call ExecuteInDefault... on each delegate passed to Given and When, as for the async versions we can just call .GetAwaiter().GetResult() in there instead of ExecuteInDefault... .

Reply:
Good catch. Since we expose the `WhenAction` directly, I've reduced the number of the calls to `ExecuteInDefault` to only the unique paths.